src/ImageSharp/Formats/Png/PngChunkType.cs

```diff
@@ -4,57 +4,57 @@
 namespace SixLabors.ImageSharp.Formats.Png
 {
     /// <summary>
-    /// Contains a list of possible chunk type identifiers.
+    /// Contains a list of of chunk types.
     /// </summary>
-    internal static class PngChunkTypes
+    internal enum PngChunkType : uint
     {
         /// <summary>
         /// The first chunk in a png file. Can only exists once. Contains
         /// common information like the width and the height of the image or
         /// the used compression method.
         /// </summary>
-        public const string Header = "IHDR";
+        Header = 1229472850U,       // IHDR
```

Comment:
I think these would be better expressed in hex so we can match them should we ever require a hex editor.